lib/benchee/formatters/html.ex

```diff
@@ -51,6 +53,12 @@ defmodule Benchee.Formatters.HTML do
   defp render_system_info(system, options \\ [visible: false]) do
     system_info(system, options)
   end
+  defp render_footer do
```

Comment:
usually we just try to put together the same methods (although the new elixir formatter will also do that differntly)

Reply:
@PragTob not sure if I understand correctly - what do you mean by "same" methods? Just above `render_footer` there are similar methods so it looks like good place to me. Could you elaborate on that?

Reply:
Oh yeah we only group the same method - as in same method and same arity. So:

```
def a do
  ...
end
def a do
  ...
end

def b do
 ...
end
def b do
  ...
end

def c do
  ...
end
```

But I think the new formatter even switches it up that all are separated by blank lines.

Sorry for the long turnaround, lots of stuff going on :)